app/jobs/spotlight/reindex_job.rb

```diff
@@ -8,7 +8,20 @@ class ReindexJob < ActiveJob::Base
       resource_list(job.arguments.first).each(&:waiting!)
     end
 
-    def perform(exhibit_or_resources)
+    around_perform do |job, block|
+      log_entry(job).reindexing_started! if log_entry(job)
```

Comment:
I can't remember if I had asked this already, but can `log_entry(job)` be stored in a variable in this block?